input/fsh/examples/participant.fsh

```diff
@@ -59,4 +72,222 @@ Description: "Example patients based on data from CBTN."
+* category.coding = http://terminology.hl7.org/CodeSystem/consentcategorycodes#research
+* policyRule = http://terminology.hl7.org/CodeSystem/consentpolicycodes#cric
+* provision.type = http://hl7.org/fhir/consent-provision-type#permit
+* provision.purpose[+] = $ncpi-data-access-code#GRU "General Research Use"
+* extension[description].valueMarkdown = "Use of the data is limited only by the terms of the model Data Use Certification."
+* extension[website].valueUrl = "https://redcap.chop.edu/surveys/?s=A7M873HMN8"
+* extension[accessType].valueCodeableConcept = $ncpi-data-access-type#controlled
+
+Instance: SD-BHJXBDQK
+InstanceOf: ResearchStudySubject
+Title: "Mock study based on kf-research-study-subject-gru-dac"
+Usage: #example
+Description: "GRU data access via DAC"
+* identifier[0]
+  * system = "https://cbtn.org/"
+  * value = "SD-BHJXBDQK"
+* title = "GRU-DAC Study Subject"
+* description = "General Research Use data access via DAC"
+* status = #completed
+* partOf = Reference(kf-research-study-cbtn)
+* extension[consent].valueReference = Reference(kf-gru-dac-consent)
+* enrollment = Reference(kf-research-study-cbtn-participants)
+
+Instance: cbtn-example-participantstudy
+InstanceOf: NcpiStudyParticipant
+Title: "Example mappings based on data from CBTN"
+Usage: #example
+Description: "Example mappings based on data from CBTN"
+* individual = Reference(PT-KZG2CZ95)  
+* study = Reference(SD-BHJXBDQK)
+* status = #candidate
+* consent = Reference(AP-GRU)  
+
+
+
+// Example 2
+
+Instance: PT-005B7CZ4
+InstanceOf: NcpiParticipant
+Title: "Example patients based on data from PCGC"
+Usage: #example
+Description: "Example patients based on data from PCGC."
+* identifier[0]
+  * system = "http://chdgenes.org/"
+  * value = "20757"
+* identifier[1]
+  * system = "http://chdgenes.org/"
+  * value = "PT_005B7CZ4"
+* birthDate.extension[+]
+  * url = $cqf-relativeDateTime
+  * extension[+]
+    * url = "target"
+    * valueReference = Reference(PT-005B7CZ4)
+  * extension[+]
+    * url = "targetPath"
+    * valueString =  "birthDate"
+  * extension[+]
+    * url = "relationship"
+    * valueCode = #after
+  * extension[+]
+    * url = "offset"
+    * valueDuration = 2006 'days'
+    * valueDuration.unit = "d"
+* extension[us-core-birth-sex].valueCode = #F "Female"
+* extension[us-core-race]
+  * extension[ombCategory].valueCoding =  $omb-race-eth#2106-3 "White"
+  * extension[text].valueString = "White"
+* extension[us-core-ethnicity]
+  * extension[ombCategory].valueCoding =  $omb-race-eth#2135-2 "Hispanic or Latino"
+  * extension[text].valueString = "Hispanic or Latino"
+
+
+Instance: pcgc-example-person
+InstanceOf: NcpiPerson
+Title: "Example patients based on data from PCGC"
+Usage: #example
+Description: "Example patients based on data from PCGC"
+* identifier
+  * system = "http://chdgenes.org/"
+  * value = "PS_123"
+* link
+  * target = Reference(PT-005B7CZ4) 
+
+Instance: AP-HMB

```

Comment:
I think they were adding AP- to these to prefix them as "Access Policy". I don't know if that really is necessary. Robert didn't suggest the curie was required here, so I think we can actually drop those few letters. I think anyone who are moderately familiar with DbGAP will recognize GRU and HBM for what they are.

Reply:
Updated the instances and references:
AP-HBM >> HBM
AP-GRU >> GRU

Reply:
Thanks!